src/read/line.rs

```diff
@@ -575,6 +582,23 @@ fn line_target_yaw(i: &str) -> IResult<VectorialStrafingConstraints> {
                 tolerance: tolerance.unwrap_or(0.),
             },
         ),
+        map(
+            preceded(
+                tag("look_at"),
+                tuple((
+                    opt(preceded(tag(" "), preceded(tag("entity "), cut(u32)))),
+                    opt(preceded(tag(" "), cut(float))),
+                    opt(preceded(tag(" "), cut(float))),
+                    opt(preceded(tag(" "), cut(float))),
```

Comment:
None of these should be optional.

Reply:
Hmmm, so the syntax should always be "target_yaw look_at entity 1 +1 -2 3"?

Reply:
No; note I commented only on the coordinates. Right now you can write `target_yaw look_at` and it will fill xyz with zeros. This should not be a thing, the coordinates are always required.